app/javascript/components/organisms/Modal.js

```diff
@@ -1,37 +1,83 @@
 import React from 'react'
 import Icon from '../atoms/Icon'
+import ReactModal from 'react-modal';
+import _ from 'lodash'
 
-const Modal = ({ header, hidden, content, primary, secondary, icon }) => {
-  return (
-    <div aria-labelledby="modalTitle" aria-hidden={hidden} role="dialog">
-      <header className="modal-inner">
-        <h1 className="modal-title t-gamma no-margin">{header}</h1>
-      </header>
-
-      <section className="modal-inner">
-        <p className="c-steel">{content}</p>
-      </section>
-
-      <footer className="modal-footer bg-dust">
-        <div className="modal-button-group row">
-          <div className="modal-button_item">
-            <button className="button primary">{primary}</button>
-          </div>
-          <div className="modal-button_item">
-            <button className="button no-border">{secondary}</button>
-          </div>
-        </div>
-      </footer>
-
-      <a className="close-reveal-modal" aria-label="Close">
-        <span className="ui-icon ui-medium i-primary">
-          <svg>
-            <use xlinkHref="#i-close"></use>
-          </svg>
-        </span>
-      </a>
+import Alert from '../organisms/Alert'
+
+const styleTypes = {
+  small: {
+    overlay: {
+      backgroundColor: 'rgba(0,0,0,0.3)'
+    },
+    content: {
+        opacity: '1',
+        border: '0',
+        borderRadius: '4px',
+        bottom: 'auto',
+        minHeight: '10rem',
+        left: '50%',
+        top: '50%',
+        padding: '0px',
+        position: 'fixed',
+        right: 'auto',
+        transform: 'translate(-50%,-50%)',
+        width: '600px'
+    }
+  }
+}
+
+const titileize = (value) =>  _.capitalize(value)
```

Comment:
Can we name this `titleize` and move it its own formatting helper file that we import so other places can also benefit from this?

Reply:
👍